humanoid_league_transform/src/humanoid_league_transform/transformer.py

```diff
@@ -97,7 +98,10 @@ def __init__(self):
         rospy.Subscriber(field_boundary_in_image_topic, PolygonStamped,
                          self._callback_field_boundary, queue_size=1)
         rospy.Subscriber(line_mask_in_image_topic, Image,
-            lambda msg: self._callback_masks(msg, self._line_mask_relative_pc_pub), queue_size=1)
+            lambda msg: self._callback_masks(
+                msg,
+                self._line_mask_relative_pc_pub,
+                scale=rospy.get_param("~masks/line_mask/scale", 1.0)), queue_size=1)
```

Comment:
I think it would be better to move the call to the parameter server outside of the lambda to avoid calling it on every single image.